### DISCUSSION TITLE
Disabling VSCode MarkDown word wrap only for hyper links?

### DISCUSSION THREAD

**Discussion by MikeLemo1** [microsoft/vscode-discussions#696] [Extension Development QnA] [answered]
Hi, 
When editing markdown documentation I come across long website links that I reference in the .md file which wrap and take quite some space,
So I'd to disable word wrap only for those marked []() Hyper links.
Is there a Vanila way to do that? if not, Is there a plugin to do that?

**Comment by burkeholland** [comment, marked as answer]
Rewrap is what you're looking for I believe: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap
